sbd_metrics.go

```diff
@@ -24,20 +24,28 @@ var (
 	}
 
 	sbdConfigPath = "/etc/sysconfig/sbd"
-	sbdPath       = "/usr/sbin/sbd"
+	sbdTools      = map[string]string{
```

Comment:
I would not add `tools` everywhere

Reply:
> thx @ashleyprimo . the `exec.LookPath` is the way to go. 
> 
> Can you describe what is the problem with the tests , without that I have to debug? I just have looked at the pr on highlevel pov

So the issue in respect of testing; is that we define tests like so:
```        corosyncTools["cfgtool"] = "test/dummy"
        corosyncTools["quorumtool"] = "test/fake_corosync-quorumtool.sh"

        _, err := NewCorosyncCollector()
```

So the issue is that now if you fed `corosyncTools["cfgtool"] = "test/dummy"` you will not be able to derive the error:
```        if err.Error() != "'test/dummy' is not executable" {
                t.Errorf("Unexpected error: %v", err)
        }
```

Since `test/dummy` script will never be found; the same can be said for `test/fake_corosync-quorumtool.sh` - this code is how much of testing is done. 

This said just thinking about this; if we could just set the $PATH environment variable to `/test` in the CI, and `corosyncTools["cfgtool"] = "test/dummy"` to `corosyncTools["cfgtool"] = "dummy"` this could work; however not ever attempted this with TravisCI - so no idea if this is possible...

Reply:
I imagine you mean, to change the wording- if so done :smiley: